src/modules/users/components/Inbox/InboxItem/InboxItem.tsx

```diff
@@ -189,7 +186,7 @@ const InboxItem = ({
             />
           </div>
         ) : (
-          <WithLink to={onClickRoute}>
+          <WithLink>
```

Comment:
If there's no link, can we remove this?

Reply:
Well, if it's not hurting anyone we can keep it as an implementation hint

Reply:
I wouldn't, as we want this back at some point.

Plus it already has the logic to just render a `div` in case no route is provided:

https://github.com/JoinColony/colonyDapp/blob/ab02a52dc58b4ef1de6860088bc49dbdb0a71fcd/src/modules/users/components/Inbox/InboxItem/InboxItem.tsx#L73-L80